root.sh

```diff
@@ -1,7 +1,7 @@
 package: ROOT
 version: "%(tag_basename)s"
-tag: "v6-26-10-alice5"
-source: https://github.com/alisw/root.git
```

Comment:
I think that we will loose a couple of commits in our fork which have not been backported to root. This should not be done.